src/main/webapp/src/app/map/map.component.ts

```diff
@@ -28,12 +30,62 @@ export class MapComponent implements AfterViewInit {
   }
 
   mapInitializer(): void {
+    
+    // Coordinates fetched from the user's location to set the center of the map.
     let centerOfMapCoordinates = new google.maps.LatLng(this.location.lat, this.location.lng);
 
     let mapOptions: google.maps.MapOptions = {
       center: centerOfMapCoordinates,
       zoom: 10
     };
     this.map = new google.maps.Map(this.mapContainer.nativeElement, mapOptions);
+
+    //Adding other markers
```

Comment:
Nit - maybe remove this comment? Might not be necessary anymore since these are the only markers, and the function name is self-explanatory.